test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,7 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
+  it("returns string with estimated time of arrival", () => {
+    expect(reachDestination(44, 10)).toEqual("I should be there in 4.5 hours.");
```

Comment:
It would be good to add a few more examples here. How do you if the function is working properly with only one test scenario?